instrumentation/github.com/gocql/gocql/observer.go

```diff
@@ -0,0 +1,263 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gocql
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/gocql/gocql"
+
+	"go.opentelemetry.io/otel/api/kv"
+	"go.opentelemetry.io/otel/api/metric"
+	"go.opentelemetry.io/otel/api/trace"
+)
+
+// OtelQueryObserver implements the gocql.QueryObserver interface
+// to provide instrumentation to gocql queries.
+type OtelQueryObserver struct {
```

Comment:
nit, OTel is an initialism and should maintain the capitalization pattern.

```suggestion
type OTelQueryObserver struct {
```